presto-spi/src/main/java/com/facebook/presto/spi/memory/LongArrayList.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.spi.memory;
+
+import java.util.Arrays;
+
+class LongArrayList
+    {
```

Comment:
wrong indentation throughout this file

What's the purpose of this class? It seems to be used only by the ArrayPool, hence, it might be better to make it a static inner class of ArrayPool.

Reply:
@mbasmanova I have moved this class to ArrayPool